README.md

```diff
@@ -138,11 +146,6 @@ ERROR: Service 'vets-api' failed to build: The command '/bin/bash --login -c bun
 make: *** [db] Error 1
 ```
 
-Sidekiq Enterprise is used for worker rate limiting and additional reliability. Most
```

Comment:
lines 141-155 should stay together.  If you move this up, move it all.

Reply:
Okay I'll move this also.